meta-lmp-base/conf/distro/include/lmp.inc

```diff
@@ -56,8 +56,7 @@ USERADD_GID_TABLES = "files/lmp-group-table"
 USERADD_ERROR_DYNAMIC = "error"
 
 # Default image formats
-IMAGE_FSTYPES = "wic wic.gz wic.bmap"
-IMAGE_FSTYPES_remove = "tar.gz tar.bz2"
+IMAGE_FSTYPES ?= "wic wic.gz wic.bmap"
```

Comment:
This one is a bit more annoying as there is no really a standard based on the BSP used, and might require a bit more work.

One example with intel-corei7-64, with the current settings:
```
# $IMAGE_FSTYPES [14 operations]
#   set? /home/rsalveti/build/lmp/build-lmp/conf/../../layers/openembedded-core/meta/conf/machine/include/x86/x86-base.inc:11
#     "wic"
#   append /home/rsalveti/build/lmp/build-lmp/conf/../../layers/meta-intel/conf/machine/intel-corei7-64.conf:33
#     "wic"
#   set /home/rsalveti/build/lmp/build-lmp/conf/../../layers/meta-lmp/meta-lmp-base/conf/distro/include/lmp.inc:60
#     "wic wic.gz wic.bmap"
...
#   append /home/rsalveti/build/lmp/build-lmp/conf/../../layers/meta-updater/classes/sota.bbclass:12
#     "${@bb.utils.contains('DISTRO_FEATURES', 'sota', 'ostreepush garagesign garagecheck ota-ext4 wic', ' ', d)}"
#   append /home/rsalveti/build/lmp/build-lmp/conf/../../layers/meta-updater/classes/sota.bbclass:13
#     "${@bb.utils.contains('BUILD_OSTREE_TARBALL', '1', 'ostree.tar.bz2', ' ', d)}"
#   append /home/rsalveti/build/lmp/build-lmp/conf/../../layers/meta-updater/classes/sota.bbclass:14
#     "${@bb.utils.contains('BUILD_OTA_TARBALL', '1', 'ota.tar.xz', ' ', d)}"
# pre-expansion value:
#   "wic wic.gz wic.bmap ${@bb.utils.contains('DISTRO_FEATURES', 'sota', 'ostreepush garagesign garagecheck ota-ext4 wic', ' ', d)} ${@bb.utils.contains('BUILD_OSTREE_TARBALL', '1', 'ostree.tar.bz2', ' ', d)} ${@bb.utils.contains('BUILD_OTA_TARBALL', '1', 'ota.tar.xz', ' ', d)}"
IMAGE_FSTYPES="wic wic.gz wic.bmap ostreepush garagesign garagecheck ota-ext4 wic   ota.tar.xz"
```

Now if we want to switch and still generate `wic.gz` and `wic.bmap`, we can't easily do that because an append would later affect initrds, and if we set per machine (in lmp-machine-custom, for example), it won't include the ostree ones, and will result in ```IMAGE_FSTYPES="wic wic.gz wic.bmap wic.vmdk wic.vdi"```, since a settings that is done per machine has higher priority and won't work with generic appends.

Similar thing happens with other machines, unfortunately. So unless we fix all the bsp layers to provide the basic and expect value at the machine config level, it won't really work with ?=.